code/game/objects/items/legcuffs.dm

```diff
@@ -92,3 +93,49 @@
 					var/mob/living/simple_animal/SA = AM
 					SA.health -= 20
 	..()
+
+
+/obj/item/restraint/legcuffs/xeno_restraints
+	name = "xeno restraints"
+	desc = "Use this to hold xenomorphic creatures moderately-safely."
+	gender = PLURAL
+	flags_atom = FPRINT|CONDUCT
+	flags_equip_slot = SLOT_WAIST
+	throwforce = 5
+	w_class = SIZE_SMALL
+	throw_speed = SPEED_SLOW
```

Comment:
will cause it to wonkily slide across the floor if you throw it, no reason to change unless you have a specific reason in mind for them to do so?